website/content/docs/api-gateway/common-errors.mdx

```diff
@@ -0,0 +1,67 @@
+---
+layout: docs
+page_title: Common Error Messages
+---
+
+# Common Error Messages
+
+Some of the errors messages commonly encountered during installation and operations of Consul API Gateway are listed below, along with suggested methods for resolving them.
+
+If the error message is not listed on this page, it may be listed on the main [Consul Common errors][consul-common-errors] page. If the error message is not listed on that page either, please consider following our general [Troubleshooting Guide][troubleshooting].
```

Comment:
```suggestion
If the error message is not listed on this page, it may be listed on the main [Consul Common errors][consul-common-errors] page. If the error message is not listed on that page either, please consider following our general [Troubleshooting Guide][troubleshooting] or reach out to in [Discuss[(https://discuss.hashicorp.com/)
```

Thoughts on maybe highlighting Discuss? Feel free to reject this suggestion if you don't feel it belongs here.